<launch>

  <node name="serial_driver_node" pkg="serial_itri_driver"  type="serial_driver_node" output="screen">
    <param name="Lport" type="string" value="/dev/ttyUSB0" />
    <param name="Rport" type="string" value="/dev/ttyUSB1" />
    <param name="baudrate" type="int" value="57600" />
  </node>

  <node name="Serial_Control" pkg="serial_itri_driver" type="Serial_Control" output="screen"/>

</launch>
